src/System.Linq.Expressions/tests/Convert/ConvertTests.cs

```diff
@@ -16710,5 +16710,103 @@ public static void PointerType()
         {
             Assert.Throws<ArgumentException>("type", () => Expression.Convert(Expression.Constant(null), typeof(int*)));
         }
+
+        public static IEnumerable<object[]> Conversions()
+        {
+            yield return new object[] { 3, 3 };
+            yield return new object[] { (byte)3, 3 };
+            yield return new object[] { 3, 3.0 };
+            yield return new object[] { 3.0, 3 };
+            yield return new object[] { 12345678, (short)24910 };
```

Comment:
This looks almost identical to the one above in ConvertCheckedTests; is there a reason this one number is different? Could these be in a shared file somewhere rather than having the duplication?